src/alarmsView/AlarmsAutoclearViewProvider.js

```diff
@@ -0,0 +1,61 @@
+import mount from 'utils/mountVueComponent';
+import AlarmsAutoclear from './AlarmsAutoclear.vue';
+import TelemetryTableConfiguration from 'openmct.tables.TelemetryTableConfiguration';
+
+export default class AlarmsAutoClearViewProvider {
+  constructor(openmct, options) {
+    this.key = 'vista.alarmsView-configuration';
+    this.name = 'Autoclear';
+
+    this.openmct = openmct;
+    this.options = options;
+    this._destroy = null;
+  }
+
+  canView(selection) {
+    if (selection.length === 0) {
+        return false;
+    }
+
+    const domainObject = selection[0][0].context.item;
+
+    return domainObject?.type === 'vista.alarmsView';
+  }
+
+  view(selection) {
+    const domainObject = selection[0][0].context.item;
+    const tableConfiguration = new TelemetryTableConfiguration(domainObject, openmct, this.options);
```

Comment:
is this one of those situations where using `this.openmct` doesn't work? let's discuss further.

Reply:
yeah, we should look at this more closely in a followup. there may be a core ticket on this. we aren't passing openmct in during initialization correctly so the plugins rely on window.openmct.